climada/hazard/trop_cyclone.py

```diff
@@ -546,14 +561,14 @@ def _close_centroids(t_lat, t_lon, centroids, buffer=CENTR_NODE_MAX_DIST_DEG):
         Mask that is True for close centroids and False for other centroids.
     """
     eye_bounds = np.c_[t_lon, t_lat, t_lon, t_lat]
-    eye_bounds[:,:2] -= buffer
-    eye_bounds[:,2:] += buffer
+    eye_bounds[:, :2] -= buffer
+    eye_bounds[:, 2:] += buffer
     centr_lat, centr_lon = centroids[:, 0], centroids[:, 1]
-    msk = (eye_bounds[:,None,1] < centr_lat[None])
-    msk &= (centr_lat[None] < eye_bounds[:,None,3])
-    msk &= (eye_bounds[:,None,0] < centr_lon[None])
-    msk &= (centr_lon[None] < eye_bounds[:,None,2])
-    return msk.any(axis=0)
+    mask = ((eye_bounds[:, None, 1] < centr_lat[None])
+            & (centr_lat[None] < eye_bounds[:, None, 3])
+            & (eye_bounds[:, None, 0] < centr_lon[None])
+            & (centr_lon[None] < eye_bounds[:, None, 2]))
+    return mask.any(axis=0)
```

Comment:
This is just for readability and a little code linting.

Reply:
Maybe a short comment to explain the mask could be useful. The hard coded indices are hard to read.